### DISCUSSION TITLE
PixArt alpha models

### DISCUSSION THREAD

**Discussion by amiraiengineer** [PixArt-alpha/PixArt-alpha#1] [General]
First congratulation PixArt-alpha team its a great project. I am searching for such a amazing project. Could you please upload the models on mentioned URL.

**Comment by yujincheng08** [comment]
working on. plz wait.

**Reply by amiraiengineer** [reply]
ok thanks

**Comment by Njasa2k** [comment]
Can't wait.

**Comment by GeorgeCazenavette** [comment]
![image](https://github.com/PixArt-alpha/PixArt-alpha/assets/18726777/46367336-c3e8-433f-ad19-00e4abee007b)

Would love to be able to include this model in my current paper; experiment freeze will be happening soon :)

**Reply by GeorgeCazenavette** [reply]
![image](https://github.com/PixArt-alpha/PixArt-alpha/assets/18726777/a3269ebf-de40-40f8-8808-c3d5126c8826)

Thanks you!

**Comment by lawrence-cj** [comment]
Model will be `released` together with Diffusers&Hugging Face team within a week, maybe

**Comment by yujincheng08** [comment]
released.
